src/App.tsx

```diff
@@ -1,61 +1,53 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { AddTodoForm } from './components/AddTodoForm';
+import { TodoList } from './components/TodoList';
+import { Todo } from './type/Todo';
+import { User } from './type/User';
```

Comment:
this is not wrong, just an improvement
so that the developer can clearly see that the import is being imported and can be written like this
`import type { User } from './type/User';`